modules/stochastic_tools/test/tests/actions/parameter_study_action/gold/cartesian_product_csv_study_samples_0001.csv

```diff
@@ -0,0 +1,26 @@
+study_sampler_0,study_sampler_1
```

Comment:
Are the lines of this file matching the lines of the qoi csv file? It would be nice to find a way to combine the two file into one.

Reply:
They are. I would say this is a separate issue that I will address in another PR. But I would have to regold it after doing so, so I might just add something to this.

Reply:
Yeah, I am fine with a separate PR. I think having the title line directly use the parameter name will be desired. Otherwise, users have to do the map in their mind.